### ISSUE TITLE
Install went fine - opens splash screen, then encounters error

### ISSUE THREAD

**Issue by hawkeye2nd** [Gictorbit/photoshopCClinux#145]
Receiving an error when running - splash screen opens, but then get a message "an unrecoverable problem has occured. photoshop will now exit."

Running Zorin 16 pro

Now - installation folder was default to C drive, not where Zorin is installed (D) but where WIndows is. Assuming that's causing the error??

**Comment by TraySSan** [comment]
Yes. If you installed it onto an NTFS partition, wine is probably having a stroke trying to make sense of it. Most likely a permission issue. Install it on an ext partition and report back.
